src/RoomsSDKAdapter.js

```diff
@@ -36,41 +39,82 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
     };
   }
 
+  /**
+   * Tells the SDK to start listening to room events and tracks the amount of calls.
+   *
+   * Note: Since the SDK listens to ALL room events, this function only
+   * calls the SDK's `rooms.listen` function on the first room to listen.
+   * Repeated calls to `rooms.listen` are not needed afterwards.
+   *
+   * @private
+   * @memberof RoomsSDKAdapter
+   */
+  startListeningToRoomUpdates() {
+    if (!this.isListening) {
```

Comment:
Can't you use `listenerCount` and remove `isListening`?
```suggestion
    if (this.listenerCount > 0) {
```

Reply:
Yep. Leftover artifact from testing code.